draft-ietf-privacypass-rate-limit-tokens.md

```diff
@@ -765,6 +770,19 @@ sec-token-request-blind = request_blind
 <Bytes containing the TokenRequest>
 ~~~
 
+### Attester behavior
+
+Upon receiving a request from a Client, the Attester first performs any required
+attestation checks of the Client, as well as checking if the Client has been
```

Comment:
```suggestion
Upon receiving a request from a Client, the Attester checks if the Client has been
```

It's possible that attestation isn't done _inline_ like this, so I think this section should just focus on the penalization behavior.